### PR TITLE
Update DSEthiop15.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Finishing

### CODE REVIEW

File: DayrSuryan/DSEthiop15.xml [hunk 1]
```diff
@@ -98,17 +100,19 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                               </item>
                            </list>
                         </collation>
-                        
-                        <condition key="good">Good condition, the leaves are slightly stained in many places.</condition>
-                        
+
+                        <condition key="good">Good condition,
+                          the leaves are slightly stained in many places. Parchment repair
+                        on <locus target="#4"/>.</condition>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop15.xml:106]: The information on the parchment repair (if it occurred during the process of parchmenting) should be included in a `<p>` in `<support>`

File: DayrSuryan/DSEthiop15.xml [hunk 2]
```diff
@@ -119,14 +123,14 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                               <dim type="left">4</dim>
                               <dim type="intercolumn">6</dim>
                            </dimensions>
-                           <ab type="ruling">Pricks are visible on <locus target="#2"/>, <locus target="#3"/>, 
+                           <ab type="ruling">Pricks are visible on <locus target="#2"/>, <locus target="#3"/>,
                            <locus target="#8"/>, <locus target="#9"/>. Ruling is visible on all leaves.</ab>
                         </layout>
-                        
+
                         <layout columns="1">
```
Comment (thea-m) [inline DayrSuryan/DSEthiop15.xml:130]: written lines?

File: DayrSuryan/DSEthiop15.xml [hunk 3]
```diff
@@ -119,14 +123,14 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                               <dim type="left">4</dim>
                               <dim type="intercolumn">6</dim>
                            </dimensions>
-                           <ab type="ruling">Pricks are visible on <locus target="#2"/>, <locus target="#3"/>, 
+                           <ab type="ruling">Pricks are visible on <locus target="#2"/>, <locus target="#3"/>,
                            <locus target="#8"/>, <locus target="#9"/>. Ruling is visible on all leaves.</ab>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop15.xml:127]: ruling pattern? is it the same as below?

File: DayrSuryan/DSEthiop15.xml [hunk 4]
```diff
@@ -136,97 +140,132 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                            <locus target="#7va"/>
                            <desc>A mistaken character has been crossed out in red.</desc>
                         </item>
-                        
+
                         <item xml:id="e11">
                            <locus target="#4"/>
                            <desc>The parchment has been mended.</desc>
                         </item>
                      </list>
```
Comment (thea-m) [inline DayrSuryan/DSEthiop15.xml:244]: This can be deleted, it is included above

File: DayrSuryan/DSEthiop15.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop15.xml:106]: Ok.

File: DayrSuryan/DSEthiop15.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop15.xml:130]: done

File: DayrSuryan/DSEthiop15.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop15.xml:244]: Done.

**Pull request merged**